tfhe/benches/integer/glwe_packing_compression.rs

```diff
@@ -1,10 +1,27 @@
 use criterion::{black_box, criterion_group, Criterion};
+#[cfg(feature = "gpu")]
+use tfhe::core_crypto::gpu::CudaStreams;
 use tfhe::integer::ciphertext::CompressedCiphertextListBuilder;
 use tfhe::integer::{ClientKey, RadixCiphertext};
+
+#[cfg(feature = "gpu")]
```

Comment:
In the integer benchmarks we added the gpu bench in a dedicated module, maybe it would be better to do the same here (see tfhe/benches/integer/bench.rs:1127).

Reply:
Done.

`bench.rs` is a little confusing to me. Please, check if it is what you have in mind.

```RUSTFLAGS="-C target-cpu=native" __TFHE_RS_BENCH_OP_FLAVOR=DEFAULT __TFHE_RS_FAST_BENCH=FALSE cargo +nightly bench --bench integer-bench --features=x86_64-unix,integer,internal-keycache,nightly-avx512,gpu -p tfhe -- integer::cuda::compression```

Reply:
@IceTDrinker Wdyt?

Reply:
I don't recall why the bench is in a separate file, but I'm fine with having the GPU in the same location if that's the question